components/Footer/Footer.module.scss

```diff
@@ -5,7 +5,7 @@
   display: flex;
   justify-content: center;
   margin-top: auto;
```

Comment:
Can we add more padding to this footer? Around 72px top and bottom should do.

![image](https://user-images.githubusercontent.com/50544486/134811134-f30015f0-8563-414c-8836-c10a7097a3b4.png)